src/app/poker-game/poker-game.component.ts

```diff
@@ -58,11 +58,15 @@ export class PokerGameComponent implements OnInit, OnChanges {
     // use api endpoint to get the delay every 30 seconds
     this.startTimer();
     // subscribe to the dev menu service
-    this.devMenuService.onMsg().subscribe((msg) => {
-      if (msg == 'TimerFinished') {
+    this.devMenuService.onMsg().subscribe((msg: DevMenuMessage) => {
+      const { message } = msg;
+      if (message == 'TimerFinished') {
```

Comment:
Man kunne overveje at lave det her til en form for Enum eller lignende for bedre language support og type safety, men altså. Det er også måske at overkomplicere det

Reply:
De forskellige beskeder er en del af `DevMenuMessage` typen, så language support her er egentlig okay :)